src/serviceconnector-passwordless/azext_serviceconnector_passwordless/_params.py

```diff
@@ -75,9 +77,11 @@ def load_arguments(self, _):
                 add_source_resource_block(c, source, enable_id=False)
                 add_target_resource_block(c, target)
                 add_auth_block(c, source, target)
+                add_configuration_store_argument(c)
                 add_new_addon_argument(c, source, target)
                 add_secret_store_argument(c)
```

Comment:
Could you please help add a new parameter to this function to align with CLI?
add_secret_store_argument(c, source)